editor/project_manager.cpp

```diff
@@ -924,10 +926,12 @@ class ProjectDialog : public ConfirmationDialog {
 		spacer->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 		default_files_container->add_child(spacer);
 
-		fdialog = memnew(FileDialog);
-		fdialog->set_access(FileDialog::ACCESS_FILESYSTEM);
-		fdialog_install = memnew(FileDialog);
-		fdialog_install->set_access(FileDialog::ACCESS_FILESYSTEM);
+		fdialog = memnew(EditorFileDialog);
+		fdialog->set_previews_enabled(false); //Crucial, otherwise the Engine crashes.
+		fdialog->set_access(EditorFileDialog::ACCESS_FILESYSTEM);
+		fdialog_install = memnew(EditorFileDialog);
+		fdialog_install->set_previews_enabled(false); //Crucial, otherwise the Engine crashes.
```

Comment:
```suggestion
		fdialog->set_previews_enabled(false); // Crucial, otherwise the engine crashes.
		fdialog->set_access(EditorFileDialog::ACCESS_FILESYSTEM);
		fdialog_install = memnew(EditorFileDialog);
		fdialog_install->set_previews_enabled(false); // Crucial, otherwise the engine crashes.
```